src/pages/Login/index.js

```diff
@@ -61,8 +78,23 @@ function Login(props) {
     }
     if (props.location.state) {
       return <Redirect to={props.location.state.from.pathname} />;
+    }
+    if (rolesList[0] === "volunteer") {
+      return <Redirect to={PATHS.CLASS} />;
+    }
+    if (rolesList[0] === "admin") {
+      return <Redirect to={PATHS.PARTNERS} />;
+    }
+    if (rolesList[0] === "partner") {
+      return <Redirect to={PATHS.PARTNERS} />;
     } else {
-      return <Redirect to={PATHS.COURSE} />;
+      return (
+        <Redirect
+          to={interpolatePath(PATHS.PATHWAY_COURSE, {
+            pathwayId: pythonPathwayId,
+          })}
+        />
+      );
     }
```

Comment:
Import a constant here and redirect, accordingly:
```
export const roleLandingPages = {
  "volunteer": PATHS.CLASS,
  "admin": PATHS.PARTNERS,
  "partner": PATHS.PARTNERS,
  "default": interpolatePath(PATHS.PATHWAY_COURSE, {
    pathwayId: pythonPathwayId,
  });
};
```
```suggestion
    return <Redirect to={rolesLandingPages[rolesList[0]] || rolesLandingPages.default} />;
```